ext/opcache/ZendAccelerator.h

```diff
@@ -310,7 +315,7 @@ extern const char *zps_api_failure_reason;
 BEGIN_EXTERN_C()
 
 void accel_shutdown(void);
-zend_result accel_activate(INIT_FUNC_ARGS);
+ZEND_RINIT_FUNCTION(zend_accelerator);
```

Comment:
Technically, this name change is not necessary, may be good to minimize the patch? I also have `accel_activate` as a name hardwired in my head to its semantics but now you actually have an other `accel_activate` function.

Reply:
`_activate` generally means the activate hook of zend_extensions, I found it even more confusing to retain it this way? Or do you prefer like `accel_activate` (the rinit) and `accel_activate_extension` (the new hook)?

Reply:
Okay, that's fair. Then I won't object against this I'll just learn the new names.